multinet/uploaders/csv.py

```diff
@@ -75,7 +72,12 @@ def upload(workspace, table):
     app.logger.info("Bulk Loading")
 
     # Read the request body into CSV format
-    body = request.data.decode("utf8")
+    try:
+        body = request.data.decode("utf8")
+    except UnicodeDecodeError:
+        response = {"errors": [{"error": "unsupported", "detail": "utf8"}]}
```

Comment:
I'm not totally sure how I feel about this, as this errors seems to imply that utf8 isn't supported, but I'm okay with it for now. This could be something we change when we standardize the REST endpoints.

Reply:
I'll replace it with `"not utf8"`